src/app/contact/contact-form/contact-form.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { FormGroup, FormControl, FormBuilder, Validators } from '@angular/forms';
+
+interface User {
+	name: string;
+	email: string;
+	Number: number;
+	input: string;
+}
+
+@Component({
+	selector: 'contact-form',
+	templateUrl: './contact-form.component.html',
+	styleUrls: ['./contact-form.component.scss']
+})
+export class ContactFormComponent implements OnInit {
+	private ContactForm: FormGroup;
+	private submitted: boolean = false;
+	@Output() private hidecontactform: EventEmitter<any> = new EventEmitter();
+
+	constructor() { }
+
+	ngOnInit() {
+		this.resetform();
+	}
+
+	private sendtosuperuser(User) {
+		console.log(User);
+	}
+
+	private resetform() {
+		this.ContactForm = new FormGroup({
+			name: new FormControl('', [<any>Validators.required]),
+			email: new FormControl('', [<any>Validators.required]),
+			Number: new FormControl('', [<any>Validators.required, <any>Validators.minLength(12)]),
```

Comment:
Should be fixed length 12? or infact country code should be selectable via a dropdown and then 10 digit telephone number. Also don't use `Number` use something like `telephone`.